src/test/cloudWatchLogs/document/logStreamDocumentProvider.test.ts

```diff
@@ -7,12 +7,16 @@ import * as assert from 'assert'
 import * as vscode from 'vscode'
 import { LogStreamDocumentProvider } from '../../../cloudWatchLogs/document/logStreamDocumentProvider'
 import { LogStreamRegistry, CloudWatchLogStreamData } from '../../../cloudWatchLogs/registry/logStreamRegistry'
+import { TestSettingsConfiguration } from '../../utilities/testSettingsConfiguration'
 
 describe('LogStreamDocumentProvider', () => {
     let registry: LogStreamRegistry
     let map: Map<string, CloudWatchLogStreamData>
     let provider: LogStreamDocumentProvider
 
+    const config = new TestSettingsConfiguration()
```

Comment:
I wonder why we named this "settings configuration". Just "settings" would do. Then we wouldn't have places in the code where it is sometimes called `settings` and sometimes `config`.

Reply:
Agreed but out of scope for this PR